src/App.js

```diff
@@ -84,6 +84,47 @@ export default function App() {
     }
   });
 
+  function success(pos) {
+    var crd = pos.coords;
+    const latitude=crd.latitude;
+    const longitude=crd.longitude;
+    var apikey = '91e34486d51b4b55b4b54c04c205ec80';
+    var api_url = 'https://api.opencagedata.com/geocode/v1/json'
+    var request_url = api_url
+      + '?'
+      + 'key=' + apikey
+      + '&q=' + encodeURIComponent(latitude + ',' + longitude)
+      + '&pretty=1'
+      + '&no_annotations=1';
+    var request = new XMLHttpRequest();
+    request.open('GET', request_url, true);
+    request.onload = function() {
+      if (request.status === 200){ 
+        var data = JSON.parse(request.responseText);
```

Comment:
Ditto. As mentioned above and everywhere else it can be replaced without leading to bugs.

Reply:
Ok Shivam , I will do the updates.
By the way I just searched for the label on Github, and got in some the initial results.